int RIGHT = 1
int LEFT = 2

int right_or_left
mediapipe_gesture_recognition/Keypoint[] keypoints

